gym/lib/gym/generators/package_command_generator_xcode7.rb

```diff
@@ -22,15 +22,14 @@ def generate
       end
 
       def options
-        options = []
+        config = Gym.config
 
+        options = []
         options << "-exportOptionsPlist '#{config_path}'"
         options << "-archivePath #{BuildCommandGenerator.archive_path.shellescape}"
         options << "-exportPath '#{temporary_output_path}'"
-
-        if Gym.config[:toolchain]
-          options << "-toolchain '#{Gym.config[:toolchain]}'"
-        end
+        options << "-toolchain '#{Gym.config[:toolchain]}'" if Gym.config[:toolchain]
```

Comment:
Cleaning up the code as you go, I love it 😍  Although you did create the `config` var above, so it might be worthwhile to replace `Gym.config` with that.